run/events-gcs/README.md

```diff
@@ -0,0 +1,93 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes GCS events
+
+## Setup
+
+Login to gcloud
+
+```sh
+gcloud auth login
+```
+
+Configure project id
+
+```sh
+gcloud config set project [PROJECT-ID]
+```
+
+Configure environment variables 
+
+```sh
+MY_RUN_SERVICE=gcs-service
+MY_RUN_CONTAINER=gcs-container
+
+MY_TOPIC=gcs-topic
+
+MY_PUBSUB_TRIGGER=pubsub-trigger
```

Comment:
Remove Pub/Sub information, `MY_TOPIC` and `MY_PUBSUB_TRIGGER` it's not needed for this sample. Just GCS.